src/prop-of.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const { flip, prop } = require('ramda');
+
+/**
+ * Returns a function that when supplied a property returns the value of this, if it exists.
```

Comment:
This doesn't actually "return a function". It returns the value of the object at the given property name.

```suggestion
 * Returns the value of `obj` at the given `propName`.
```